J/jemalloc/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "jemalloc"
+version = v"5.2.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/jemalloc/jemalloc.git", "886e40bb339ec1358a5ff2a52fdb782ca66461cb")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd jemalloc/
+autoconf
+
+if [[ "${target}" == "x86_64-linux-gnu" ]]; then
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-initial-exec-tls
```

Comment:
`x86_64-linux-gnu` isn't any special, apart from the fact it's the platform on which CI is running and as such the only platform for which we can actually dlopen the library.  If `--disable-initial-exec-tls` is needed to dlopen the library on this platform, chances are that this is needed everywhere, it's just that here we see the problem only on the host platform.  In #932 this was done indeed for all platforms.  See also https://github.com/JuliaPackaging/Yggdrasil/pull/932#issuecomment-643710732 and message below (which repeats what I just wrote here)